src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -273,8 +291,11 @@ internal static void CheckForSevereException(Exception e)
             }
             finally
             {
-                TelemetryAPI.ReportExitTelemetry(s_theConsoleHost);
-                s_theConsoleHost.Dispose();
+                if (s_theConsoleHost != null)
```

Comment:
I presume System.Console APIs won't work if there is no process console.  However, I can use the ConsoleHost class tracer to write errors.  But I don't feel that reporting a command line parse error in this case is very important.